### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by jandrieu** [WebOfTrustInfo/vRWOT_SkotOS#2]
We need to create a base for documentation, starting with how we log in and growing to cover how we interact with and program the system.

**Comment by wip-abramson** [comment]
Yep definitely agree. I am not oriented in this code at all. A getting started guide for developers would be great. If someone (@noahgibbs maybe) could onboard me I would be happy to take on the responsibility to write this process up into docs others could follow.

**Comment by noahgibbs** [comment]
I could definitely get you started. What time zone are you in? (I'm UK-based.)

There's also information in https://ChatTheatre.github.io/SkotOS-Doc that could be useful, but it's not a fantastic starting point (yet.) Working on it!

**Comment by wip-abramson** [comment]
Great, let me have a pass through the docs and then get back to you. I am also UK based so hopefully shouldn't be too hard to sync up if needed.
